### ISSUE TITLE
physics shape is zero sized for a CCNode whose width is percentage of its parent

### ISSUE THREAD

**Issue by buptwsg** [spritebuilder/SpriteBuilder#839]
How to reproduce:
1. drag a Color Node to scene
2. set the node's width to 80% of its parent container
3. go to the third tab on the right panel, enable physics
4. observe the physics shape, it's wrong

this bug is not critical, but it's nice to fix it.

**Comment by cocojoe** [comment]
Agreed, the quick fix is to quickly switch your content size type to a fixed value type and then enable physics.
